frontend/src/pages/register.js

```diff
@@ -0,0 +1,212 @@
+    if (password !== passwordVerification) {
+      this.setState({ showInvalidPasswordModal: true })
+    } else {
+      registerUser(email, password, 'member').then(resp => {
+        if (resp.status === 400) {
+          this.setState({ showInvalidRequestModal: true })
+        } else {
+          localStorage.setItem('interviewerKey', memberKey) // TODO: Create switch statements for roles
+          Router.push('/dashboard')
+        }
+      })
+    }
+  }
+
+  handleInvalidPasswordModalClose = () => {
+    this.setState({ showInvalidPasswordModal: false })
+  }
+
+  handleInvalidRequestModalClose = () => {
+    this.setState({ showInvalidRequestModal: false })
+  }
+
+  handleInvalidGoogleRequestModalClose = () => {
+    this.setState({ showInvalidGoogleRequestModal: false })
+  }
+
+  render() {
+    return (
+      <>
+        <Head />
+        <Nav />
+        <Container>
+          <Card className="interview-card" style={{ width: '400px', height: '60%' }}>
+            <CardTitle>
+              <h3 style={{ textAlign: 'center', paddingTop: '10px' }}>Register</h3>
+            </CardTitle>
+            <CardBody>
+              <Form>
+                <FormGroup>
+                  <Label for="exampleEmail">Email</Label>
+                  <Input
+                    type="email"
+                    name="email"
+                    maxLength="64"
+                    pattern={EMAIL_REGEX}
+                    value={this.state.email}
+                    onChange={this.handleChange}
+                    required
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="examplePassword">Password</Label>
+                  <Input
+                    type="password"
+                    name="password"
+                    minLength="8"
+                    maxLength="64"
+                    value={this.state.password}
+                    onChange={this.handleChange}
+                    required
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="examplePassword">Confirm Password</Label>
+                  <Input
+                    type="password"
+                    name="passwordVerification"
+                    minLength="8"
+                    maxLength="64"
+                    value={this.state.passwordVerification}
+                    onChange={this.handleChange}
+                    required
+                  />
+                </FormGroup>
+                <Button
+                  color="success"
+                  size="lg"
+                  onClick={this.handleSubmit}
+                  style={{ float: 'left', width: '48%' }}
+                >
+                  Register
+                </Button>
+              </Form>
+              <GoogleLogin
+                className="btn-lg sign-in-btn"
+                clientId="992779657352-2te3be0na925rtkt8kt8vc1f8tiph5oh.apps.googleusercontent.com"
```

Comment:
re: my comment about storing this kind of info in a config

Reply:
(you can handle this in a follow-up PR so we can ship this one faster)

Reply:
#326

Reply:
Change to `'409847273934-jmhjkeu77d3cqr32sh3vpl3ogh2f4dev.apps.googleusercontent.com'`